radar/2021-01-01/tailwindcss.md

```diff
@@ -0,0 +1,25 @@
+---
+title:      "Tailwind CSS"
+ring:       trial
+quadrant:   languages-and-frameworks
+
+---
+Tailwind CSS is a utility first css framework which heavily utilizes css classes. What seems a very "different" approach
+in the beginning turns into a big "ah-ha-moment" during development and even more during build step. The classes
+utilized are entirely generated based on a free to edit configuration file which outlines for example the entire design
+system including states, nuances and so forth. Tailwinds high flexibility results in a very ux/design aligned set of css
+classes with a fraction of writing code compared to a self built solution. For example colors, sizes, spaces etc. can
+have meaningful names that are easy to remember and shared between devs and designers. This of course results in a
+shared language with lesser explanation required. To sum the positive experience up Tailwind CSS has furthermore a deep
+integration into the dev & build processes which means, during built time it's recognized which parts are really
+necessary. This obviously lead to very small build sizes with a nearly 100% css coverage.
+
+The deep integration and the extraordinary small build sizes were main aspects for us to choose Tailwind CSS for
+resource limited industry 4.0 projects. These projects have a huge demand on a variety of interaction forms.
+
+Tailwind helps us fast-forward to fulfil modern user expectations by reducing complexity of sophisticated industry
+processes in a multi device approach.
+
+This is what users can expect nowadays in industry processes as well because the evolution from e.g. a classic cellular
+phone towards a smartphone has proven, there is still a huge potential for improvements towards usability and adaptivity
+driven industry applications.
```

Comment:
Tried to simplify and improve clarity:

```suggestion
Tailwind CSS is a framework that heavily utilizes CSS classes.
What seems to be a very different approach in the beginning, turns into a big "ah-ha-moment" during development and even more during the build step.
CSS classes are entirely generated based on a configuration file that outlines the entire design system including states, nuances, etc.
Tailwinds high flexibility results in a set of CSS classes aligned with UX/design, requiring just a fraction of code compared to a self-built solution.
Colours, sizes, spaces etc. can have meaningful names that are easy to remember and shared between developers and designers.
In turn, this results in a shared language with less explanation required.
Support for deep integration into the development and build processes ensure optimized build times with incremental rebuilds only on parts really necessary.
This obviously leads to very small build sizes with nearly 100% CSS coverage.

The deep integration and the extraordinary small build sizes were the main aspects for us to choose Tailwind CSS for resource-limited projects in the area of industry 4.0.
These projects have a huge demand on a variety of interaction forms.

Tailwind helps us fast-forward to fulfil modern user expectations by reducing the complexity of sophisticated industrial processes in a multi-device approach.

This is what users can expect nowadays in industrial processes as well because the evolution from e.g. classic cellular phones towards smartphones has proven, there is still a huge potential for improvements in usability and adaptivity industry applications.
```